tools/bin/config.js

```diff
@@ -42,6 +42,10 @@ const externals = [{
   name: 'angularfire',
   entry: 'angularfire',
   globalName: 'angular'
+}, {
+  name: 'd3',
+  entry: 'd3',
+  globalName: 'd3'
 }];
```

Comment:
This array is use to build the options for `jspm build`, do define the libraries to exclude to define how the find them on the global scope.

See https://github.com/singpath/example-app/blob/master/docs/jspm.md#production-build